types/coin_test.go

```diff
@@ -17,9 +17,9 @@ func TestIsPositiveCoin(t *testing.T) {
 		{NewCoin("a", -1), false},
 	}
 
```

Comment:
Same with the CHANGELOG. Not sure if this diff is supposed to be part of this PR?